bucket/simple-dnscrypt.json

```diff
@@ -0,0 +1,62 @@
+{
+    "version": "0.7.1",
+    "description": "An easy to use GUI for dnscrypt-proxy",
+    "homepage": "https://simplednscrypt.org/",
+    "license": "MIT",
+    "suggest": {
+        "vcredist2017": "extras/vcredist2017"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/bitbeans/SimpleDnsCrypt/releases/download/0.7.1/SimpleDNSCrypt64.msi",
+            "hash": "5BA0C94774ACD7ADE18446B4351E7D28A948C490AEFC50DC6C08C13551F19F7F",
+            "post_install": [
+                "if (Test-Path \"$dir\\SimpleDNSCrypt64.msi\") {Remove-Item \"$dir\\SimpleDNSCrypt64.msi\" | Out-Null}"
+            ]
+        },
+        "32bit": {
+            "url": "https://github.com/bitbeans/SimpleDnsCrypt/releases/download/0.7.1/SimpleDNSCrypt.msi",
+            "hash": "CAC6B8119AF3DA70A5977A2E73EE5214303990FEDD7ABBC5F247A1E2CF19A663",
+            "post_install": [
+                "if (Test-Path \"$dir\\SimpleDNSCrypt.msi\") {Remove-Item \"$dir\\SimpleDNSCrypt.msi\" | Out-Null}"
+            ]
+        }
+    },
+    "pre_install": [
+        "if (!(is_admin)) { error \"$app requires admin rights to $cmd\"; break }",
+        "",
+        "if (!(Test-Path \"$env:LOCALAPPDATA\\SimpleDnsCrypt\")) {New-Item -Path \"$env:LOCALAPPDATA\" -Name \"SimpleDnsCrypt\" -ItemType \"directory\" -Force | Out-Null}"
+    ],
```

Comment:
```suggestion
```

I tried installing version 0.7.0, then update to version 0.7.1, and the config  is being tranferred without a problem, so I think we can just let app handle it.

Reply:
The reason I have it there is because on my end, the app fails to start without this folder in appdata/local.

Reply:
Okay, Let's leave this command, but I don't think we need **admin rights** to create new folders in `$env:LOCALAPPDATA`

Reply:
Ok I just tested it, it turns out you're right. I will get rid of the `(!(is_admin))` command.